.github/workflows/ci.yml

```diff
@@ -234,6 +339,58 @@ jobs:
         echo "LLVM_BUILD_DIR=$LLVM_BUILD_DIR" >> $GITHUB_ENV
         echo "CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH" >> $GITHUB_ENV
 
+    - name: Build and Test/Install CppInterOp on macOS
```

Comment:
Likewise.

Reply:
Yeah merged them, just a thing to be mentioned is lld linker is not supported on macOS. Hence i removed that flag and now default ld64 linker is being used in every environment. Is this change ok for our project??

Reply:
This can be solved by adding a matrix switch to handle the platform-specific cmake options: https://github.com/vgvassilev/clad/blob/59fd672f54d171577857fe75ed9223b91d7902d1/.github/workflows/ci.yml#L93